xarray/core/dataset.py

```diff
@@ -1849,7 +1849,7 @@ def to_zarr(
 
         See Also
         --------
-        http://xarray.pydata.org/en/stable/io.html#zarr
+        :doc:`io.zarr`
```

Comment:
This gave `WARNING: unknown document: io.zarr`. It should work with `:ref:` instead of `:doc:` I believe...

Reply:
Thanks @zmoon. Apologies, i'm by lazy here and didn't build the docs. You want to make a PR to my branch?

Reply:
@raybellwaves I am also lazy, I saw the message in the rtd log. I don't know why it still says pending, but it seems to have failed-on-warning after 10 minutes.